src/Http/Headers/src/CacheControlHeaderValue.cs

```diff
@@ -652,7 +652,7 @@ private static void AppendValues(StringBuilder sb, IEnumerable<StringSegment> va
                     sb.Append(", ");
                 }
 
-                sb.Append(value);
+                sb.Append(value.AsSpan());
```

Comment:
For my edification, what is the benefit of StringBuilder.Append with a Span over a String?

Reply:
value isn't a string, it's a StringSegment that has implicit converters to Span and Memory. In 3.0 Append added Span and Memory overloads and now we get ambiguity errors.